doc/source/rllib/rllib-algorithms.rst

```diff
@@ -893,7 +899,7 @@ often: The "forward" model will eventually get better at predicting these next l
 
 
 
-.. _RE3:
+.. _re3:
```

Comment:
what do we gain from this? did you check that still works?

Reply:
Checked it works.  Sphinx documentation said ref links are supposed to be lower case..